MeetupPlatformApi/Features/Meetups/SignUpForMeetup/SignUpForMeetupFeature.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace MeetupPlatformApi.Features.Meetups.SignUpForMeetup;
+
+using AutoMapper;
+using MeetupPlatformApi.Authentication.Helpers;
+using MeetupPlatformApi.Persistence.Context;
+using MeetupPlatformApi.Seedwork.WebApi;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+[ApiSection(ApiSections.Meetups)]
+public class SignUpForMeetupFeature : FeatureBase
+{
+    private readonly ApplicationContext context;
+    private readonly IMapper mapper;
+
+    public SignUpForMeetupFeature(ApplicationContext context, IMapper mapper)
+    {
+        this.context = context;
+        this.mapper = mapper;
+    }
+
+    /// <summary>
+    /// Sign up for meetup by its id.
+    /// </summary>
+    /// <response code="404">If needed meetup is null.</response>
+    /// <response code="400">If user doesn't exist or the user is trying to sign up for meetup twice.</response>
+    /// <response code="200">If there's not errors after executing method.</response>
+    [HttpPost("/api/meetups/sign-up/{id:guid}")]
+    [Authorize(Roles = Roles.PlainUser)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(typeof(MeetupInfoDto), StatusCodes.Status200OK)]
+    public async Task<IActionResult> SignUpForMeetup([FromRoute] Guid id)
```

Comment:
In this method You have used an `.Include` for both `meetup` and `user`. Please, remove one of these includes and use the other one